src/index.html

```diff
@@ -1,13 +1,22 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    
+    <div class="container">
+      <div class="container__block container__block--1"></div>
+      <div class="container__block container__block--2"></div>
+      <div class="container__block container__block--3"></div>
+      <div class="container__block container__block--4"></div>
+      <div class="container__block container__block--5"></div>
+      <div class="container__block container__block--6"></div>
```

Comment:
no, don't use modifiers for `n-th` elements, use pseudoclasses intead
```suggestion
      <div class="container__block"></div>
      <div class="container__block"></div>
      <div class="container__block"></div>
      <div class="container__block"></div>
      <div class="container__block"></div>
      <div class="container__block"></div>
```

Reply:
Have I changed it right?